addons/project/models/project_task.py

```diff
@@ -158,6 +159,7 @@ def _read_group_personal_stage_type_ids(self, stages, domain, order):
         *CLOSED_STATES.items(),
         ('04_waiting_normal', 'Waiting'),
     ], string='State', copy=False, default='01_in_progress', required=True, compute='_compute_state', inverse='_inverse_state', readonly=False, store=True, index=True, recursive=True, tracking=True)
+    is_closed = fields.Boolean("Closed state", store=False, compute='_compute_is_closed', search='_search_is_closed')
```

Comment:
FYI, when we add a compute method in the field definition then the field will be non-stored and readonly. 🙂 

```suggestion
    is_closed = fields.Boolean("Closed state", compute='_compute_is_closed', search='_search_is_closed')
```

See https://github.com/odoo/odoo/blob/master/odoo/fields.py#L434-L442